docs/reference/all_strategies.rst

```diff
@@ -102,6 +102,9 @@ Here are the docstrings of all the strategies in the library.
 .. automodule:: axelrod.strategies.mathematicalconstants
    :members:
    :undoc-members:
+.. automodule:: axelrod.strategies.memorydecay
```

Comment:
There is no need for this as the strategy is not in this module but in `axelrod.strategies.meta` (which is already included).